modules/oe_translation_epoetry/src/Form/ModifyLinguisticRequestForm.php

```diff
@@ -99,9 +99,13 @@ public static function access(TranslationRequestEpoetryInterface $translation_re
       return AccessResult::forbidden()->addCacheableDependency($cache);
     }
 
-    // We only allow to add new languages if the request is accepted. The API
-    // allows other statuses as well but we try to simplify.
-    $allowed = $translation_request->getEpoetryRequestStatus() === TranslationRequestEpoetryInterface::STATUS_REQUEST_ACCEPTED && $translation_request->getRequestStatus() === TranslationRequestEpoetryInterface::STATUS_REQUEST_ACTIVE;
+    // We only allow to add new languages if the request is accepted or
+    // suspended.
+    $states = [
+      TranslationRequestEpoetryInterface::STATUS_REQUEST_ACCEPTED,
+      TranslationRequestEpoetryInterface::STATUS_REQUEST_SUSPENDED,
```

Comment:
I decided to add also Suspended to the allowed list of statuses because it's similar to active/ongoing, but temporarily paused. So actions can be performed in this status. i discussed with DGT. I also adjusted the tests to account for this.